LIPs/LIP-governable_security_committee.md

```diff
@@ -0,0 +1,41 @@
+---
+lip: governable_security_committee
```

Comment:
I don't think there is sufficient information in this LIP for it to be merged as a draft yet so I recommend fleshing out the empty sections before submitting a PR to merge it as a draft.